app/controller/partycontroller.ctrl.php

```diff
@@ -218,7 +218,7 @@ public function create()
                                        "<p><strong>Party End Time:</strong> ". $end ." </p>" .
                                        "<p> Thank you!!! <p>";
 
-                              $subject = APPNAME . ": Party created by the host : " . $hostnamep . " ";
+                              $subject = APPNAME . ": Party created by the host : " . $hostname . " ";
                               $headers = "From: " . APPEMAIL . "\r\n";
                               $headers .= "Content-Type: text/html; charset=ISO-8859-1\r\n";
                               $email= NOTIFICATION_EMAIL;
```

Comment:
Could you add this new constant in to config.example.php, with a made up email address.